docs/spark/README.md

```diff
@@ -0,0 +1,20 @@
+## Data Privacy Toolkit Spark Wrappers
+
+What follows describes the spark wrappers available to leverage the functionality of **DPT** in a Spark environment.
+
+The Spark JAR was built for Java 11 and Spark 3.2.3.
```

Comment:
is this still accurate?

Reply:
Yes, Spark version has been improved to the stable one for 3.2 but Java is maintained to 11 for consistency with previous engagements.